shared/cluster.go

```diff
@@ -409,3 +409,139 @@ func WriteDataPod(namespace string) (string, error) {
 
 	return RunCommandHost(cmd)
 }
+
+type ServiceType string
+
+const (
+	Server ServiceType = "server"
```

Comment:
Suggestion: Using `nodeType` instead of `serviceType`?

Reply:
fixed and moved to product.go